cli/pub_command.go

```diff
@@ -66,7 +67,8 @@ Available template functions are:
 	pub := app.Command("publish", pubHelp).Alias("pub").Action(c.publish)
 	pub.Arg("subject", "Subject to subscribe to").Required().StringVar(&c.subject)
 	pub.Arg("body", "Message body").Default("!nil!").StringVar(&c.body)
-	pub.Flag("wait", "Wait for a reply from a service").Short('w').BoolVar(&c.req)
+	pub.Flag("wait", "Wait for a single reply from a service").Short('w').BoolVar(&c.req)
+	pub.Flag("wait-multiple", "Wait for multiple replies from services. 0 waits until timeout").Default("1").IntVar(&c.waitCount)
```

Comment:
Wouldnt it be more logical if `nats req` just had an option to wait for multiple responses or a set period keeping `nats pub` as sending only? Having `nats pub` gain some `req` features that `req` doesnt have seems weird to me. 

While these 2 just reuse the same logic and is so close together the line between publish only and interact with services is pretty strong so would want to keep that

Rest of the PR is great thank you

Reply:
Yes I have the delta code in tbe other function I mentioned. If we need to go all in and do that might still be worth making that a bit more generic for reuse

Reply:
@ColinSullivan1 The `--wait-multiple` option on `nats pub` was elegant.  What are you thinking for `nats req` -- something like just `--multiple`?

Reply:
Don’t think delta based early termination makes sense here. It makes sense for server api interactions like varz because we know servers respond immediately. We do not know that for some random service.

Reply:
I went ahead and added some very simple delta-like code upon request, and that behavior can be overridden with `--request 0`.  Certainly open for discussion or suggestions.  Also moved to the sync sub - it was simpler.

Reply:
I updated the description to reflect the new usage, and the timestamps give an indication of UX.